assets/js/template-helpers/isEmptyNode.js

```diff
@@ -0,0 +1,19 @@
+export default function (node, options) {
+  var nodeItems = [
+    node.types,
+    node.functions,
+    node.macros,
+    node.callbacks,
+    node.headers
+  ].filter(Array.isArray);
```

Comment:
Wouldn't a simpler implementation instead traverse this list of nodeItems and then check that they are null or an empty list? This way we don't need to flatten nor build a very large list only to check its length later.

Reply:
Fair point @josevalim. Refactored to iterate through nodeItems and return true as soon as it detects content. Otherwise, false.